src/epcpm/parameterstointerface.py

```diff
@@ -1315,26 +1313,19 @@ def create_item(
                 sunspec_type,
             ]
 
-            node_in_model = get_sunspec_point_from_table_element(
-                sunspec_point=sunspec2_point,
-                table_element=table_element,
+            maybe_model = sunspec2_point.tree_parent
```

Comment:
Umm what's a `maybe_model`?

Reply:
`Maybe` you'd like to know?

Reply:
Refactor in c66ddfe93252752340a94e0fe4495d484132ca90

Reply:
:p